code-challenges.test.js

```diff
@@ -20,11 +30,34 @@ const fibLength1 = 6
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+
+
+const result = [0, 1]; 
+for(let i = 2; i < 0; i++) {
+  const a = result[i - 1];
+  const b = result[i - 2];
+  result.push(a + b);
+}
+return result;
+
 // b) Create the function that makes the test pass.
 
+const fibLength1Result = fibonacciArray(6);
+exportAllDeclaration(fibLength1Result).toEqual([0, 1, 1, 2, 3, 5]);
+
+const fibLength2Result = fibonacciArray(10);
+exportAllDeclaration(fibLength1Result).toEqual([0, 1, 1, 2, 3, 5, 8, 13, 21, 34])
+
+
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
+// Psuedo code:
+// create the object as an argument, take the values from the object using `object.values()`
+// Sort through the values from least to greatest using the `array.sort()` method that will return the sorted value.
+// Create an array as an arguement, return the array where each element is the sum of all elements before it in the input array.
+ 
```

Comment:
Nice pseudocode! I think you are heading in the right direction! Where is the function?